src/app/clusters/scenes-server/SceneTable.h

```diff
@@ -203,8 +209,7 @@ class SceneTable
 
         void Clear()
         {
-            SetName(CharSpan());
-
+            memset(mName, 0, sizeof(mName));
```

Comment:
SetName(CharSpan{}) was better as it handled the mNameLength.

Reply:
Agree, reverted.